test-helpers/db-connection.js

```diff
@@ -1,7 +1,7 @@
 const neo4j = require('neo4j-driver').v1;
 
 const driver = neo4j.driver(
-	process.env.NEO4J_BOLT_URL,
+	process.env.NEO4J_BOLT_URL || 'bolt://localhost:7687',
```

Comment:
once the tasks are in make, you will get the env vars for free and not need this

Reply:
this is not working locally despite putting the tasks in make ... 
```
Uncaught TypeError: Bolt URL expected to be string but was: undefined
```

Reply:
err, ok, leave it as it is then